py3-cython.yaml

```diff
@@ -0,0 +1,58 @@
+package:
+  name: py3-cython
+  version: 3.0.10
+  epoch: 1
+  description: Cython is an optimising static compiler for both the Python & the extended Cython programming languages.
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    provider-priority: 0
+
+vars:
+  pypi-package: cython
+
+data:
+  - name: py-versions
+    items:
+      3.10: "310"
+      3.11: "311"
+      3.12: "312"
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - py3-supported-pip
+      - py3-supported-python-dev
+      - py3-supported-wheel
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha256: 00f97476cef9fcd9a89f9d2a49be3b518e1a74b91f377fe08c97fcb44bc0f7d7
+      uri: https://github.com/cython/cython/archive/${{package.version}}.tar.gz
+
+subpackages:
+  - range: py-versions
+    name: py${{range.key}}-${{vars.pypi-package}}
+    dependencies:
+      provider-priority: ${{range.value}}
+      provides:
+        - ${{package.name}}
+        - cyphon
+    pipeline:
+      - uses: py/pip-build-install
+        with:
+          python: python${{range.key}}
+      - uses: strip
+
+  - name: py3-wheels-${{vars.pypi-package}}
+    pipeline:
+      - uses: py/collect-wheels
```

Comment:
@smoser note, no py3-supported-cython as they are not co-installable (due to /usr/bin/cython) and also not splitting that up, because cython is a compiler that happens to be written in python; i don't believe anything imports Cython modules. If they do we can split up the bins later.

Unless you want to split and provide coinstallable Cython module today like I do in  pip & wheel.